server/sonar-web/src/main/js/components/SourceViewer/components/SCMPopup.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import { SourceLine } from '../../../app/types';
+import BubblePopup from '../../common/BubblePopup';
+import DateFormatter from '../../intl/DateFormatter';
+
+interface Props {
+  line: SourceLine;
+  popupPosition?: any;
+}
+
+export default function SCMPopup({ line, popupPosition }: Props) {
+  return (
+    <BubblePopup customClass="source-viewer-bubble-popup" position={popupPosition}>
+      <div className="bubble-popup-section">{line.scmAuthor}</div>
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([typescript:S1192](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1192))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fstas%2Fpopups&issues=AWHb6qakw6XGHKvPuQTR&open=AWHb6qakw6XGHKvPuQTR)

[ISSUE_KEY]: AWHb6qakw6XGHKvPuQTR